src/client/stylesheets/_gmr-results.scss

```diff
@@ -0,0 +1,95 @@
+@use "govuk-frontend" as *;
+@use "variables" as *;
+
+.gmr {
+  .govuk-summary-list__key,
+  .govuk-summary-list__value {
+    font-size: 16px;
```

Comment:
use the `govuk-font` mixin to set the font-size everywhere in this file
https://design-system.service.gov.uk/styles/type-scale/